### ISSUE TITLE
Any chance of help integrating with pyqt app?

### ISSUE THREAD

**Issue by matt-seaton** [s4w3d0ff/python-poloniex#104]
This isn't really an issue with the code as it works great (thanks btw) but I'm trying to build a  GUI app and I can't get any of the push stuff working.  In fact it seems the onJoin callback never gets executed.  Would anybody happen to have any ideas how to get it to work?  I followed this guy's example to integrate the pyqt event loop with quamash and switched to autobahn.asyncio:

https://github.com/icefo/PyQt5-Wamp/blob/master/PyQt5_GUI.py

That was pretty straightforward and it works fine except for the callback problem.  My QMainWindow also inherits from ApplicationSession so it should fire, right?  It's weird because onConnect and onDisconnect both work as expected.  If I call it manually in init with self.onJoin() it just doesn't do anything.  I put a print statement in onJoin() and... nothing.   

You can see my code in the issues section of his page (it's the only one).

**Comment by matt-seaton** [comment]
OK I have a relevant issue now.  I changed the code just slightly and now every chat message gets printed out twice, although the logger formatting is different for the 2nd message.  The only things I changed were the following:

autobahn.asyncio instead of autobahn.twisted 
removed the inline callback decorator and used the async keyword instead
changed yield to await
imported unescape from html instead of the deprecated usage

Any ideas?
